### ISSUE TITLE
[MacOS] Apple Silicon build

### ISSUE THREAD

**Issue by artur9010** [ankermake/AnkerMake-PrusaSlicer-Release#23]
Currently ankermake studio is provided as x86 Mac app, it would be nice to have a build for apple silicon

<img width="1152" alt="Screenshot 2024-06-01 at 20 06 24" src="https://github.com/ankermake/AnkerMake-PrusaSlicer-Release/assets/4835677/2537f30d-4d85-4ded-86e3-69e1f67d24c1">

**Comment by rttnbrcht** [comment]
I’m waiting for this too. This would probably also improve performance. Prusaslicer and orcaslicer both have native macOS arm versions for a while now.

**Comment by FlavorFx** [comment]
Me too. The startup time is much longer on macOS arm CPUs compared to old machines.

**Comment by npjohnson** [comment]
This is absolutely needed before macOS release next year.

In short, we need an arm64 build (or universal binary) version instead of just an Intel x86_64 version.

@AnkerXuan @Anker-louisdu can you please confirm if this is in the roadmap?

This is critical to the product continuing to work on macOS.

**Comment by FlavorFx** [comment]
Well, a port to arm64 will not solve the problem, because the open source version does not contain the network stack. The stack in to open source. This means the printer can't be accessed by e.g. eufy studio or app.
I contacted the support from eufy two months ago to make this problem transparent. This printer will be useless to Mac user. Unfortunately, there is no final answer yet.

**Comment by npjohnson** [comment]
@FlavorFx Yeah, of course they'd need to update all dependencies to support arm64... that's assumed in saying "an arm64 port"

**Comment by FlavorFx** [comment]
I spoke with eufy support. They will neither continue to support the macOS version nor will they open-source the network stack. This means the community won't be able to port the source code with the network connectivity to macOS arm64. This makes the 3D printer useless to me.
